src/models/deployers/SubscriptionsParamsManager.js

```diff
@@ -38,30 +38,30 @@ module.exports = class extends BaseDeployer {
     const agentCallsScript = []
 
     // fee token and amount
-    if (this.feeToken) {
-      this._addActionToCallsScript(agentCallsScript, aragonNetworkDAO.agent, this._setFeeToken(subscriptions.address, subscriptions.feeToken, subscriptions.feeAmount))
+    if (this.feeToken && this.feeAmount) {
```

Comment:
Why is this? Changing the fee token assumes changing the fee amount (in the contract setter, I mean), so I don't see the need for this.

Reply:
Sure it does, but with this, I'm requesting the admin to give both params, otherwise we were assuming the admin wanted the same fee amount for another token which doesn't sound realistic

Reply:
See, discussion below. The misunderstanding here is that to me these were just boolean flags, not numeric values.

Reply:
I think you miss-placed your answer here

Reply:
Why? If these are not numeric values, but booleans, you don't need two of them.

>the admin wanted the same fee amount for another token which doesn't sound realistic

Btw, this could perfectly happen if the change is for instance from SAI to DAI, or from USDT to DAI, etc.

Reply:
To clarify further, the idea was 1 flag -> 1 setter.